content/tracing/advanced_usage/_index.md

```diff
@@ -279,6 +279,88 @@ if (($span = GlobalTracer::get()->getActiveSpan()) !== null){
 {{% /tab %}}
 {{< /tabs >}}
 
+## Change Agent Hostname
```

Comment:
Not sure, but in the preview I don't see this section in the navigation menu (on the left)